framework/src/modules/http_api/helpers/utils.js

```diff
@@ -139,12 +139,32 @@ const filterByParams = (peers, filters) => {
 const consolidatePeers = (connectedPeers = [], disconnectedPeers = []) => {
 	// Assign state 2 to the connected peers
 	const connectedList = [...connectedPeers].map(peer => {
-		const { ipAddress, options, minVersion, nethash, ...peerWithoutIp } = peer;
+		const {
+			ipAddress,
+			options,
+			minVersion,
+			nethash,
+			nonce,
+			blockVersion,
+			maxHeightPrevoted,
+			lastBlockId,
+			...peerWithoutIp
```

Comment:
Does this var name still name sense? It will be the peer object without all the above properties and not only `ip`